src/platforms/react-native/performance/instrumentation/user-interaction-instrumentation.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: User Interaction Instrumentation
+sidebar_order: 20
+sdk: sentry.javascript.react-native
+
+description: "Learn what kind of user interaction can be captured"
+---
+
+The UI instrumentation, once enabled, captures transactions and adds breadcrumbs for touch interactions. Gesture support using React Native Gesture Handler is available using the `traceGesture` wrapper. 
+
+## Touch Events
+
+The SDK composes the transaction name from the name of a displayed screen (for example `LoginScreen`) and a label of the element user interacted with (for example `login_button`).
+
+If the element doesn't have `sentry-label` or a user-defined label, the transaction won't be captured.
```

Comment:
This is also mentioned on line 42. Should this be a note?

Reply:
I've removed the duplicity. I don't think it needs to be a note but for sure it should not repeat.